src/translation/english.c

```diff
@@ -1130,6 +1131,19 @@ static translation_string all_strings[] = {
     {TR_PARAMETER_VALUE_BUILDING_MENU_PARKS, "All decorative parks"},
     {TR_PARAMETER_VALUE_BUILDING_MENU_SMALL_TEMPLES, "All small temples"},
     {TR_PARAMETER_VALUE_BUILDING_MENU_LARGE_TEMPLES, "All large temples"},
+    {TR_PARAMETER_VALUE_BUILDING_NATIVE_CROPS, "Native crops"},
+    {TR_PARAMETER_VALUE_BUILDING_SMALL_TEMPLE_CERES, "Small Ceres temple"},
+    {TR_PARAMETER_VALUE_BUILDING_SMALL_TEMPLE_NEPTUNE, "Small Neptune temple"},
+    {TR_PARAMETER_VALUE_BUILDING_SMALL_TEMPLE_MERCURY, "Small Mercury temple"},
+    {TR_PARAMETER_VALUE_BUILDING_SMALL_TEMPLE_MARS, "Small Mars temple"},
+    {TR_PARAMETER_VALUE_BUILDING_SMALL_TEMPLE_VENUS, "Small Venus temple"},
+    {TR_PARAMETER_VALUE_BUILDING_LARGE_TEMPLE_CERES, "Large Ceres temple"},
+    {TR_PARAMETER_VALUE_BUILDING_LARGE_TEMPLE_NEPTUNE, "Large Neptune temple"},
+    {TR_PARAMETER_VALUE_BUILDING_LARGE_TEMPLE_MERCURY, "Large Mercury temple"},
+    {TR_PARAMETER_VALUE_BUILDING_LARGE_TEMPLE_MARS, "Large Mars temple"},
+    {TR_PARAMETER_VALUE_BUILDING_LARGE_TEMPLE_VENUS, "Large Venus temple"},
+    {TR_PARAMETER_VALUE_BUILDING_SENATE_UPGRADED, "Senate upgraded"},
+    {TR_PARAMETER_VALUE_BUILDING_FORUM_UPGRADED, "Forum upgraded"},
```

Comment:
Upgraded senate and forum don't really exist, so I don't think you need these strings.

Reply:
Done.